tests/pyccel/test_pyccel.py

```diff
@@ -0,0 +1,39 @@
+import subprocess
```

Comment:
@EmilyBourne Codacy complains about a few things in this file. Can you please have a look at it?

Reply:
Codacy doesn't like the use of subprocess which I use to run pyccel via pytest. I can't think of any other way to do this in python. The alternative would be a bash script but ideally that would require a bash equivalent of pytest. Do you have any suggestions?

Reply:
Yes, there is probably no way around it. But before we give up on this, have you tried using `shell=False`?

Reply:
I've managed to remove the shell=True, but it doesn't seem to be enough to satisfy codacy

Reply:
Thanks for trying! The warnings raised by Codacy are less serious now (notice the orange highlight instead of the red we had previously), and your code looks good to me.

@ratnania: Can we reduce the strictness level of our Codacy analysis?

Reply:
@EmilyBourne: While we wait for Ahmed's input we can skip rule [B603](https://docs.openstack.org/bandit/latest/plugins/b603_subprocess_without_shell_equals_true.html) in file [bandit.yml](https://github.com/pyccel/pyccel/blob/master/bandit.yml).